### PR TITLE
Create a Recommendation File for Brasilia

### PR DESCRIPTION
**Pull request by luizgglopes**
Things to do:

- [x] Add commit
- [x] Review comment
- [x] Official review
- [x] Merge

### CODE REVIEW

**Comment by jongrim** [comment]
👍  🌮

**Comment by natalia-souza** [comment]
hahahaha

File: luizguilhermegl-brasilia.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Places to eat
+- Pastel
+- Churrascaria
+
+### Places to visit
+- JK Bridge
+
+### Places to avoid
+- Danger places
```
Comment (jongrim) [inline luizguilhermegl-brasilia.md]: Here comes danger 🔪

Comment (Xukinorris) [inline luizguilhermegl-brasilia.md]: Quais são os lugares perigosos hein mané? 😛

**Review by jongrim — APPROVED** [review]
Excellent

**Pull request merged**